src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<Fruit, Integer> recordsMap = new HashMap<>();
+
+    public static Map<Fruit, Integer> getAll() {
+        return recordsMap;
+    }
+
+    public static Integer get(Fruit fruit) {
+        return recordsMap.get(fruit);
+    }
+
+    public static String addPair(Fruit fruit, Integer quantity) {
+        recordsMap.put(fruit, quantity);
+        return "Succesfully added pair {" + fruit.name() + ", " + quantity + "}";
```

Comment:
Usually, such messages are sent to log files, but not as return value, so you can remove it.